scripts/llext_link_helper.py

```diff
@@ -32,56 +33,107 @@ def parse_args():
 def main():
 	parse_args()
 
+	TYPE_NONE = 0
+	TYPE_DATA = 1
+	TYPE_BSS = 2
+	last_type = TYPE_NONE
+
+	SHF_ALLOC = 2
+	SHF_EXECINSTR = 4
```

Comment:
A simple `git grep` showed that these are already defined, no need to duplicate

```
ipython3
In [1]: from elftools.elf.constants import SH_FLAGS

In [2]: SH_FLAGS.SHF_EXECINSTR
Out[2]: 4
```

Reply:
> In [1]: from elftools.elf.constants import SH_FLAGS

that's the line I couldn't find, thanks